homeassistant/components/waqi/sensor.py

```diff
@@ -68,104 +73,129 @@ async def async_setup_platform(
     station_filter = config.get(CONF_STATIONS)
     locations = config[CONF_LOCATIONS]
 
+    async_create_issue(
```

Comment:
I wonder if this shouldn't be moved to `config_flow` so it's only if it's a success and there seems to be an issue missing if they have something in YAML that can't be imported e.g. station missing or something?

Reply:
Well currently you fill in a search query and some specifics to match the found stations better. If that doesn't return any stations, I would still like them to know that they should remove the yaml but it's not my problem that their query is wrong.

Reply:
Sure but it could maybe be a different issue telling them the query returned nothing so they need to act upon it something more than just removing the YAML